test/tests_storage.js

```diff
@@ -266,7 +267,41 @@ describe("Storage Tests", () => {
         // will set to default storage path
         storage.setStoragePath("../test/customStorageDirectory/", true);
         assert.equal(storage.getStoragePath(), "../test/customStorageDirectory/");
-        recordValuesToStorageAndValidate();
+        recordValuesToStorageAndValidate("../test/customStorageDirectory/", false, true);
+        done();
+    });
+
+    it("14- Setting storage path to default path via initStorage /no-init", (done) => {
+        storage.resetStorage();
+        storage.initStorage();
+        assert.equal(storage.getStoragePath(), "../data/");
+        done();
+    });
+
+    it("15- Setting bulk storage path to default path via initStorage /no-init", (done) => {
+        storage.resetStorage();
+        storage.initStorage(null, false, true);
+        assert.equal(storage.getStoragePath(), "../bulk_data/");
+        done();
+    });
+
+    it("16- Setting custom storage path via initStorage /no-init", (done) => {
+        storage.resetStorage();
+        storage.initStorage("../test/customStorageDirectory/");
+        assert.equal(storage.getStoragePath(), "../test/customStorageDirectory/");
+        done();
+    });
+
+    it("17- Setting storage method to memory only and checking storage path /no-init", (done) => {
+        storage.resetStorage();
+        storage.initStorage(null, cc.storageTypeEnums.MEMORY);
+        assert.equal(storage.getStoragePath(), undefined);
+        done();
+    });
+
+    it("18- Recording to storage with memory only storage /no-init", (done) => {
+        storage.resetStorage();
+        recordValuesToStorageAndValidate(null, cc.storageTypeEnums.MEMORY);
```

Comment:
we need SDK running with different storage config options. not testing the module internally. so a real usecase, like initial tests above.